### PR TITLE
Create Adi'sBadAtPhysics

### PR DESCRIPTION
**Pull request by Kepler45**
lmao

### CODE REVIEW

File: Adi'sBadAtPhysics [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+                        if (i + length < numeral.Count())
+                        {
+                            if (numeral[i] == numeral[i + length])
+                                length++;
+                            else
+                                repeat = false;
+                        }
+                        else
+                            repeat = false;
+
```
Comment (Peter-Baker-SPS) [inline Adi'sBadAtPhysics:87]: ioj,chjkfuig;;tghj,.uwZ

File: Adi'sBadAtPhysics [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+                            repeat = false;
+    
+                    
+                    numout.Add(intToNumeral(length));
+                    numout.Add(numeral[i].ToString());
+                    i += length - 1;
+                    length = 1;
+                }
+                num = string.Join("", numout);
+                numout.Clear();
```
Comment (Peter-Baker-SPS) [inline Adi'sBadAtPhysics:95]: ;H;HK;H;

**Review by Peter-Baker-SPS — CHANGES_REQUESTED** [review]
cHANGE evErYthIng ahahHAHAHhahAHahAHahAHahAHahAHahAHah

**Comment by Kepler45** [comment]
ar yu ok petr

**Comment by Kepler45** [comment]
image

**Pull request closed without merging**